Bowling/Bowling/Frames.cs

```diff
@@ -0,0 +1,286 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BowlingApp
+{
+    public class Frames
+    {
+        public const int frameNumberTotal = 10;   // total number of frames in a bowling game
+        public const int bonusFrame = 10;    // frame where bonus roll could occur
+        private List<Frames> framesCollection = new List<Frames>();  // the collection of Frames objects
+        private int marksLength = 0;  // provides the total length of bowling scores string
+        private int placeCount = 0;  // placeCount will be used as a pointer in our bowling score string
+        private string score = string.Empty;  // string for each scoring mark in the bowling score string
+
+        public int frameNumber { get; set; }  // the frame number for these rolls
+        public int firstRoll { get; set; }  // first roll score
```

Comment:
We may be leaking some business logic here (re: encapsulation).  I would investigate which of these setters could benefit from being `private`.